src/generated/cloud/bigquery/admin/v2/.sidekick.toml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[general]
+specification-source = 'google/cloud/bigquery/v2'
+service-config = 'google/cloud/bigquery/v2/bigquery_v2.yaml'
+
+[source]
+skipped-ids = """
+.google.cloud.bigquery.v2.JobService.Query,\
+.google.cloud.bigquery.v2.PostQueryRequest,\
+.google.cloud.bigquery.v2.QueryRequest,\
+.google.cloud.bigquery.v2.QueryResponse,\
+.google.cloud.bigquery.v2.JobService.GetQueryResults,\
+.google.cloud.bigquery.v2.GetQueryResultsRequest,\
+.google.cloud.bigquery.v2.GetQueryResultsResponse"""
+
+[codec]
+copyright-year = '2025'
+package-name-override = 'google-cloud-bigquery-admin-v2'
+not-for-publication = 'true'
```

Comment:
Why?

Reply:
Because I am not sure if we want to remove more RPCs before publishing.